### PR TITLE
Jersey version upgraded to 2.28

### PR DESCRIPTION
**Pull request by senivam**
Signed-off-by: Maxim Nesen <maxim.nesen@oracle.com>

### CODE REVIEW

**Review by Cousjava — APPROVED** [review]

**Comment by m0mus** [comment]
@Cousjava Why are merged this PR without waiting until all checks are green??? Have you seen that the build failed???
https://jenkins.eclipse.org/glassfish/blue/organizations/jenkins/glassfish/detail/PR-22716/1/pipeline
Please next time wait until all checks are green before doing merge.

**Pull request merged**
